scripts/admin_clean_provider_machine.py

```diff
@@ -117,7 +117,7 @@ def find_kernel_ramdisk(accounts, machine_request):
             print " Fail! Old Image:%s ALSO missing kernel+ramdisk" % old_glance_image
         try:
             existing_parent_request = MachineRequest.objects.get(
-                new_machine__identifier=image_id)
+                new_machine__identifier=machine_request.parent_image_id)
```

Comment:
machine request has both a new_image_id and a parent_image_id - but no image_id.  Since the variable name that we are setting is named existing_parent_request, I am assuming that we want the parent_image_id.  It seems logical.